website/docs/integrations/vscode.mdx

```diff
@@ -22,72 +23,125 @@ description: ConfigCat Visual Studio Code extension. This is a step-by-step guid
 - Connect a Config to your Workspace.
 - Open a Config on ConfigCat Dashboard.
 
+<img src="/docs/assets/vscode/ff_setting_view.gif" className="zoomable" alt="Usage of ConfigCat Feature Flags Visual Studio Code Extension" />
+
 ## Install extension
 
-### Visual Studio Code Marketplace
+### Install from Visual Studio Code Marketplace
 
 1. Open [ConfigCat Feature Flags Extension](https://marketplace.visualstudio.com/items?itemName=ConfigCat.configcat-feature-flags) at Visual Studio Marketplace.
 1. Click on the Install button.
-1. Configure extension (see below)
+1. [Configure extension](#configure-extension).
 
-### or within Visual Studio Code
+### Install within Visual Studio Code
 
-1. Search for ConfigCat Feature Flags at the Extensions page.
+1. Search for ConfigCat Feature Flags on the Extensions page.
```

Comment:
Javaslom, hogy az első pontban mindig nevezzük meg a toolt (bár látom, hogy itt a cím sokat segít).

Valami ilyet javaslok:
In Visual Studio Code, open the Extensions page, and search for ConfigCat Feature Flags.